### PR TITLE
Gql02 user get by id ap itest murzabayev

### PR DESCRIPTION
**Pull request by AzamatMurzabay**
-userCreate
-userDeleteById
-userGetById
- userGetAll
- userUpdateById

### CODE REVIEW

**Review by Jalalov01 — CHANGES_REQUESTED** [review]

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                query: userCreateM,
+                variables: userInput,
+            };
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body;
+                    userId = res.body.data.userCreate._id;
+                    console.log("RESP BODY ===", respData);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:26]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    console.log("USER ID ===", userId);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:27]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            const postData = {
+                query: userGetByIdQ,
+                variables: userGet
+            }
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body.data;
+                    console.log("RESP BODY USER GET BY ID ===", respData);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:44]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                query: userCreateM,
+                variables: userInput,
+            };
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body;
+                    userId = res.body.data.userCreate._id;
+                    console.log("RESP BODY ===", respData);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:69]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+            }
+            const postData = {
+                query: userGetByIdQ,
+                variables: userGet
+            }
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body.data;
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:86]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+                    console.log("RESP BODY USER GET BY ID ===", respData);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:87]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+                variables: userInput,
+            };
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body;
+                    userId = res.body.data.userCreate._id;
+                    console.log("RESP BODY ===", respData);
+                    console.log("USER ID ===", userId);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:70]: done

File: src/modules/user/_test/userGetByIdMur.spec.js [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+            }
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body.data;
+                    console.log("RESP BODY USER GET BY ID ===", respData);
+                    // expect (respData.userGetById._id).eq(userId);
+                    // expect (respData.userGetById.firstName).eq(userInput.userInput.firstName);
+                    // expect (respData.userGetById.lastName).eq(userInput.userInput.lastName);
```
[the remark this answers is not in the document]
Comment (AzamatMurzabay) [reply src/modules/user/_test/userGetByIdMur.spec.js:90]: done

**Comment by Jalalov01** [comment]
All changes verified

**Review by Jalalov01 — APPROVED** [review]

**Pull request merged**